components/multiselect/multiselect_list.tsx

```diff
@@ -1,17 +1,46 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
 import Constants from 'utils/constants.jsx';
 import {cmdOrCtrlPressed} from 'utils/utils.jsx';
 import LoadingScreen from 'components/loading_screen.jsx';
+
+import {Props as MultiSelectProps, State as MultiSelectState, Value} from './multiselect';
+
+type Props = Pick<
```

Comment:
I think this use of the `Pick` utility type here is a bit overkill, is something that is not going to change a lot and I think we don't need to express that in this way here, I think we can just have a bit of duplication to have more readability.

I would suggest here just use plain types, what do you think @devinbinnie?

Reply:
My thinking for using `Pick` was to make sure it stayed consistent with the interface defined in the other component. Happy to change it if that's the preference.